src/systems/systemItemsDialog.component.tsx

```diff
@@ -79,7 +79,9 @@ const SystemItemsDialog = React.memo((props: SystemItemsDialogProps) => {
   return (
     <Dialog
       open={open}
-      onClose={(reason) => reason == 'backdropClick' ?? onClose}
+      onClose={(event, reason) =>
+        event && reason == 'backdropClick' ? null : onClose()
+      }
```

Comment:
```suggestion
      onClose={(_event, reason) =>
        reason !== 'backdropClick' && handleClose()
      }
```
Handle close is present above.

Reply:
It isn't used for the cancel button, shall I change the button to use handleClose ?

Reply:
Yes